eng/helix/content/RunTests/ProcessUtil.cs

```diff
@@ -59,6 +59,11 @@ public static Task CaptureDumpAsync(int pid, string dumpFilePath)
                 return Task.CompletedTask;
             }
 
+            if (!File.Exists($"{Environment.GetEnvironmentVariable("HELIX_WORKITEM_ROOT")}/dotnet-dump"))
```

Comment:
What about the Windows case i.e. where `dotnet-dump.exe` exists instead❔
```suggestion
            if (!File.Exists($"{Environment.GetEnvironmentVariable("HELIX_WORKITEM_ROOT")}/dotnet-dump") &&
                !File.Exists($"{Environment.GetEnvironmentVariable("HELIX_WORKITEM_ROOT")}/dotnet-dump.exe"))
```